### ISSUE TITLE
appName option in ringo/httpserver is confusing

### ISSUE THREAD

**Issue by botic** [ringo/ringojs#207]
``` javascript
var httpServer = new Server({
    "appName": "app",
    "appModule": module.resolve("./actions"),
    "port": 8080
});
```

The option `appName` suggests to be the application's name, which is not the case. Indeed it's the exported property name of the JSGI application!

**Comment by grob** [comment]
Would it be too restrictive to get rid of the misleading `appModule`/`appName` combination by enforcing the use of `module.exports` in the module that exports the JSGI application function?

This way the `app` option would be either function or a string taken as path to a module - in the latter case the module would be expected to use `module.exports` to export the stick Application instance (or a plain JSGI function).

Thinking along that path, we could even change the signature of the `Server` constructor to `Server(<String | Function>[, options])`, making a clearer distinction between the required JSGI function and optional parameters.

**Comment by botic** [comment]
I updated the documentation and added an example for the current state.

**Comment by botic** [comment]
Closed for #358
